src/Compilers/RealParserTests/RandomRealParserTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
The tests run for about 60 seconds, and caught all of the bugs that I had to fix. There is no mechanism to run it longer, except by running it again.